packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -4,11 +4,12 @@
   "description": "generates a Courage version for Sign-In Widget",
   "main": "src/CourageForSigninWidget.js",
   "scripts": {
-    "build": "scripts/build.sh"
+    "build": "webpack"
   },
   "devDependencies": {
-    "@okta/babel-plugin-handlebars-inline-precompile": "3.1.0-alpha.3586.gd86894f",
-    "@okta/courage": "4.6.0-alpha.3586.gd86894f",
+    "@okta/babel-plugin-handlebars-inline-precompile": "3.1.0-alpha.3660.g8faa6ac",
```

Comment:
This module `packages/@okta/courage-for-signin-widget/` getting more responsibility rather than just pulling Courage.
Some further refactoring seems necessary. (addressed separately) 
One idea:
- rename to more general module name like `okta-internal-modules`
- add script per module like
    - `build-courage`
    - `build-handlebar-inline-precompile`
    - `build-eslint-rule`
    - `build` compose all above (and apparently can be done in parallel)